cmd/plasmacli/config/config.go

```diff
@@ -3,66 +3,118 @@ package config
 import (
 	"bytes"
 	"fmt"
+	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/spf13/viper"
 	cmn "github.com/tendermint/tendermint/libs/common"
 	"os"
 	"path/filepath"
+	"strings"
 	"text/template"
 )
 
 const defaultConfigTemplate = `# This is a TOML config file.
 # For more information, see https://github.com/toml-lang/toml
 
-##### ethereum config options #####
+##### ethereum configuration #####
+
 # Ethereum plasma contract address
-ethereum_plasma_contract_address = "{{ .EthPlasmaContractAddr }}"
+ethereum_contract_address = "{{ .EthPlasmaContractAddr }}"
 
 # Node URL for eth client
 ethereum_nodeurl = "{{ .EthNodeURL }}"
 
-# Number of Ethereum blocks until a transaction is considered final
-ethereum_finality = "{{ .EthBlockFinality }}"`
+
+##### plasamd configuration #####
+
+# Node URL for plasmad
+node = "{{ .PlasmadNodeURL }}"
+
+# Trust the connected plasmad node (don't verify proofs for responses)
+trust_node = {{ .PlasmadTrustNode }}
+
+# Chain identifier. Must be set if trust-node == false
+chain_id = "{{ .PlasmadChainID }}"`
 
 // Must match the above defaultConfigTemplate
-type PlasmaConfig struct {
-	EthPlasmaContractAddr string `mapstructure:"ethereum_plasma_contract_address"`
+type Config struct {
+	// Ethereum config
+	EthPlasmaContractAddr string `mapstructure:"ethereum_contract_address"`
 	EthNodeURL            string `mapstructure:"ethereum_nodeurl"`
-	EthBlockFinality      string `mapstructure:"ethereum_finality"`
```

Comment:
Why was this part of config removed??

Reply:
It was never used in the cli

Reply:
Doesn't this get parsed and used in `eth`? Or is there a separate `Finality` parameter in some other config file that gets used?

Reply:
that parameter is set to 0 in the CLI

Reply:
this should be opened as an issue if it already isn't. CLI needs to support finality bounds